rpc_storage_handler.go

```diff
@@ -596,31 +596,8 @@ func (r *RPCStorageHandler) SetRollingWindow(keyName string, per int64, val stri
 
 }
 
-// SetScrollingWindow is used in the rate limiter to handle rate limits fairly.
 func (r *RPCStorageHandler) SetRollingWindowPipeline(keyName string, per int64, val string) (int, []interface{}) {
```

Comment:
I tried this first, but it doesn't work at all. I forgot to mention why in the commit, will update.